support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -178,9 +178,9 @@ export const tests: Tests = {
 			},
 		],
 		audiences: {
-			GBPCountries: {
+			ALL: {
```

Comment:
This will pick-up AUS too, is the Link button working for `GNM Membership AUS` in PROD? If not we should only include regions using `GNM Membership`

Reply:
It's not enabled in the payment method settings so it won't show up

Reply:
But we'll still be assigning users in Australia to the A/B test despite users in the `variant` never seeing the button (as it's not enabled), so that could skew the results against the `variant`, the impact in reality will probably be negligible however if we know certain users will never experience the variant experience then we should exclude them from the test.